src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {
+    BLACK, WHITE, BLUE, GREEN, YELLOW, RED, BROWN, ORANGE, PINK

```

Comment:
all entries in enum must be from new line

Reply:
Why? 
An enum class with no methods and no documentation on its constants may optionally be formatted as if it were an array initializer (see Section 4.8.3.1 on [array initializers](https://google.github.io/styleguide/javaguide.html#s4.8.3.1-array-initializers)).

private enum Suit { CLUBS, HEARTS, SPADES, DIAMONDS }